packages/react/src/toast-action/ToastAction.tsx

```diff
@@ -18,7 +18,7 @@ export const ToastAction = forwardRef<HTMLButtonElement, ToastProps>(
           orientation="vertical"
         />
 
-        <Button asChild size="sm" variant="solid" {...props}>
+        <Button appearance="secondary" asChild size="sm" {...props}>
```

Comment:
let's add a new appearance for these called `inverse`

so `inverse` will use `colorScheme="neutral"` and `variant="solid"`

Reply:
should we show it in storybook bhai?

Reply:
Nah we don’t need to show it there - the toast stories will cover them